lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/handler/DrivingPermitHandler.java

```diff
@@ -325,13 +328,20 @@ private IdentityVerificationService createIdentityVerificationService(
     }
 
     private ThirdPartyAPIService selectThirdPartyAPIService(
-            boolean dvaDirectEnabled, String documentCheckingRouteHeader) {
-        // TODO Change to a switch for each api and read feature flag + header
-        if (dvaDirectEnabled && "dva-direct".equalsIgnoreCase(documentCheckingRouteHeader)) {
-            // DVA
+            boolean dvaDirectEnabled,
+            boolean dvlaDirectEnabled,
+            String documentCheckingRoute,
+            String licenseIssuer) {
+
+        IssuingAuthority issuingAuthority = IssuingAuthority.valueOf(licenseIssuer);
+
+        boolean direct = "direct".equals(documentCheckingRoute);
+
+        if (direct && (issuingAuthority == IssuingAuthority.DVA) && dvaDirectEnabled) {
```

Comment:
I think this might be worth having 2 headers for as otherwise in the lower envs we will always be hitting DVLA before its ready if the header has been set

Reply:
We could avoid two headers by toggling the API DVA/DVLA feature flags, turning them on one at a time as they are ready for api tests - as both the header route and feature flag is needed this would prevent hitting the in-development code.